families-api/app/router.py

```diff
@@ -254,7 +255,25 @@ class GeographyDocumentCount(SQLModel):
 )
 def docs_by_geo(
     session: Session = Depends(get_session),
+    corpus_import_ids: list[str] = Query(
+        default=[],
+        alias="corpus.import_id",
+    ),
+    document_statuses: list[str] = Query(
+        default=[],
+        alias="document.status",
```

Comment:
Given that the field is [`document.document_status`](https://github.com/climatepolicyradar/navigator-backend/blob/82e7d710c954e4a2dc1c942a320297e33469ceed/families-api/app/models.py#L397) - should we consider using that to avoid having too many levels of transformation?

Reply:
Trying to avoid the [semantic gap](https://climate-policy-radar.slack.com/archives/C062K5MMPMH/p1756285409085769?thread_ts=1756285309.795719&cid=C062K5MMPMH) (a term I learnt recently)